frontend/src/model/ConvertDataFormat.js

```diff
@@ -7,8 +7,7 @@ export function convertToDataObject(databaseDevice) {
     databaseDevice.device.ipAddress,
     databaseDevice.device.displayName,
     databaseDevice.device.status,
-    databaseDevice.inputs,
-    ["Additional Device Info goes here."]
```

Comment:
We should still have an extras/notes section, but the database doesn't have that field yet.

Reply:
The concern you pose is part of a greater issue that needs to be fixed in coordination with the backend in another PR. These fields do not match the data representation being sent between service and frontend, and I "labelled" them undefined to highlight this fact.

Reply:
Created issue #184 to reflect the problem with this ConvertDataFormat function and its tests.